llvm/test/tools/append-file/append-file.test

```diff
@@ -0,0 +1,6 @@
+-- Verify append-file creates the expected output file
+-- Append files
+RUN: append-file %S/Inputs/a.txt --append=%S/Inputs/b.txt --output=%t.txt
+
+-- Verify result
+RUN: diff %t.txt %S/Inputs/ab.txt
```

Comment:
If the input files are minimized to 1 line per file, `ab.txt` could be removed in favor of:
```suggestion
RUN: FileCheck %s --input-file %t.txt
CHECK: aaaaaaaaaaa
CHECK-NEXT: bbbbbbbbbbb
```

Reply:
As a follow-up thought, simple `RUN: echo aaaaaaaa > %s.a.txt` commands would allow to discard the `Inputs` altogether. I'm personally fine with either approach.

Reply:
Thanks for the suggestions!  I've updated the test accordingly.